ember.js

```diff
@@ -31,14 +32,20 @@ module.exports = {
         "config/**/*.js",
         "tests/dummy/config/**/*.js",
         "lib/*/index.js",
-        "server/**/*.js"
+        "server/**/*.js",
+        "dist/**",
+        "tmp/**",
+        "node_modules/**"
       ],
       excludedFiles: [
         "addon/**",
         "addon-test-support/**",
         "app/**",
         "tests/dummy/app/**"
       ],
+      rules: {
+        'compat/compat': false
```

Comment:
The plugin is also activate on the override part?

Reply:
It disables the linting plugin on unwanted files such as `node_modules`. I wonder how other plugins avoid running on such files. Any idea @GreatWizard?

Reply:
linting was not enabled for node_modules, but you add them in the files list in this PR 🤔

Reply:
I push a tiny refactor in that commit to be able to deactivate compat on defined files: https://github.com/peopledoc/eslint-config-peopledoc/pull/41/commits/521db504c364abe18515d4d9ac67891acf1f07a8

Reply:
it seems ok our projects. We need now to add he good exclusions settings (Polyfill enabled in our apps)